src/recovery/cubic.rs

```diff
@@ -162,6 +182,29 @@ fn on_packet_acked(
 
     if in_congestion_recovery {
         return;
+    } else {
+        // Detecting spurious congestion events.
+        // <https://tools.ietf.org/id/draft-ietf-tcpm-rfc8312bis-00.html#section-4.9>
```

Comment:
This doesn't actually explain how to detect the spurious event, where does the algorithm come from?

Reply:
The draft doesn't specify a detailed algorithm how to detect it. This is my own.